fr/integrations/plugin-packs/procedures/network-dell-os10-snmp.md

```diff
@@ -0,0 +1,204 @@
+| load5                       | System load 5 minutes sample     |      |
+| load15                      | System load 15 minutes sample    |      |
+
+<!--Memory-->
+
+| Metric name             | Description                | Unit |
+| :---------------------- | :------------------------- | :--- |
+| memory.usage.bytes      | Memory usage               | B    |
+| memory.free.bytes       | Free memory                | B    |
+| memory.usage.percentage | Memory usage in percentage | %    |
+| memory.buffer.bytes     | Buffer memory              | B    |  
+| memory.cached.bytes     | Memory cached              | B    |
+| memory.shared.bytes     | Shared Memory allocation   | B    |
+
+<!--Swap-->
+
+| Metric name                 | Description   | Unit |
+| :-------------------------- | :------------ | :--- |
+| swap.usage.bytes            | Swap usage    | B    |
+| swap.free.bytes             | Swap free     | B    |
+| swap.usage.percentage       | Swap usage    | %    |
+
+<!--Uptime-->
+
+| Metric name                 | Description                                        | Unit |
+| :-------------------------- | :------------------------------------------------- | :--- |
+| uptime                      | Duration of system has been working and available. | s    |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+Afin de contrôler vos équipements Dell OS10, le SNMP v2 doit être configuré.
+Pour plus d'information, vous pouvez vous référer à la documentation utilisateur officiel :
+https://www.dell.com/support/manuals/fr/fr/frbsdt1/networking-z9100/os10-enterprise-user-guide-10_4_2-pub/configure-snmp?guid=guid-cf75b1b6-5c24-4237-af0c-b9ed1f491b75&lang=en-us
+
+## Installation
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Dell OS10 :
+
+```bash
+yum install centreon-plugin-Network-Dell-Os10-Snmp.noarch
+```
+
+2. Sur l'interface Web de Centreon, installer le Plugin-Pack *Dell OS10 SNMP* depuis la page "Configuration > Plugin packs > Manager"
+
+<!--Offline IMP License-->
+
+1. Installer le Plugin sur tous les collecteurs Centreon supervisant des ressources Dell OS10 :
+
+```bash
+yum install centreon-plugin-Network-Dell-Os10-Snmp.noarch
+```
+
+2. Sur le serveur Central Centreon, installer le Plugin-Pack via le RPM:
+
+```bash
+yum install centreon-pack-network-dell-os10-snmp.noarch
+```
+
+3. Sur l'interface Web de Centreon, installer le Plugin-Pack *Dell OS10 SNMP* depuis la page "Configuration > Plugin packs > Manager"
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+Ce Plugin-Pack est conçu de manière à avoir dans Centreon un hôte par équipement Dell OS10.
+Lorsque vous ajoutez un hôte à Centreon, appliquez-lui le modèle Net-Dell-Os10-SNMP-custom. 
+Il est nécessaire de remplir les valeurs des champs "SNMP Community" et "SNMP Version".
+Une fois celui-ci configuré, certaines macros doivent être renseignées:
```

Comment:
Cette ligne n'est pas nécessaire pour du SNMP

Reply:
Done. It removed